src/app/collective-rewards/rewards/backers/hooks/useGetRBI.ts

```diff
@@ -51,25 +51,27 @@ export const useGetBackerRBI = (backer: Address, { rbtc, rif }: Record<string, T
     const accumulatedAllocationsTime = gauges_.reduce(
       (acc, { accumulatedAllocationsTime_, allocation_, lastBlockTimestamp_: gaugeLastBlockTimestamp_ }) => {
         const lastStakedSeconds = Big(timestamp.toString()).sub(gaugeLastBlockTimestamp_)
-        return acc.add(accumulatedAllocationsTime_).add(Big(allocation_).mul(lastStakedSeconds))
+        return acc.add(accumulatedAllocationsTime_).add(lastStakedSeconds.mul(allocation_))
       },
       Big(0),
     )
 
+    const rifPrice = prices[rif.symbol]?.price ?? 0
+
+    const priceAdjustedAllocTime = accumulatedAllocationsTime.div(WeiPerEther.toString()).mul(rifPrice)
+
+    if (priceAdjustedAllocTime.eq(0)) {
+      return Big(0)
+    }
+
```

Comment:
```suggestion
    const rifPrice = prices[rif.symbol]?.price ?? 0

    if (rifPrice.eq(0) || accumulatedAllocationsTime.eq(0)) {
      return 0n
    }
    
    const priceAdjustedAllocTime = accumulatedAllocationsTime.div(WeiPerEther.toString()).mul(rifPrice)

```

no need to do the calculation if it's going to end up being 0